docs/contracts/libraries.rst

```diff
@@ -30,9 +30,10 @@ not possible to destroy a library.
 Libraries can be seen as implicit base contracts of the contracts that use them.
 They will not be explicitly visible in the inheritance hierarchy, but calls
 to library functions look just like calls to functions of explicit base
-contracts (``L.f()`` if ``L`` is the name of the library). Furthermore,
-``internal`` functions of libraries are visible in all contracts, just as
-if the library were a base contract. Of course, calls to internal functions
+contracts (using qualified access like ``L.f()``). However,
```

Comment:
I don't understand what you mean my the sentence that starts with `However,` - I would just remove it.